src/Kanbas/Database/quizzes.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
Missing description, assignmentGroupId, quizTypeId, and accessCode.

Reply:
I used enums for assignmentGroup and quizType. AccessCode and Description added.